ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +570,63 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    explicit_paddings=[],
+    use_cudnn_on_gpu=False,
+    data_format='NHWC',
+    dilations=[1, 1, 1, 1],
+    name="Conv2D"
+
+):
+    # ivy.backends.tensorflow expects strides and dilations to be
+    # a single integer value or a list of 2 values whereas the raw op
+    # expects a list of 4 values
+    if data_format == "NHWC":
+        strides = strides[1:-1]
+        dilations = dilations[1:-1]
+    elif data_format == "NCHW":
+        strides = strides[2:]
+        dilations = dilations[2:]
+
+    # for Conv2D, the explicit_padding is defined as
+    # [[0, 0], [pad_top, pad_bottom], [pad_left, pad_right], [0, 0]]
+    # when the data_format is "NHWC"
+    # for Conv2D, the explicit_paddings is defined as
+    # [[0, 0], [0, 0], [pad_top, pad_bottom], [pad_left, pad_right],
+    # when the data_format is "NCHW"
+    if padding == "EXPLICIT" and data_format == "NHWC" :
+        padding = explicit_paddings[1:2]
+        return ivy.conv2d(
+            input,
+            filter,
+            strides,
+            padding,
+            explicit_paddings,
```

Comment:
if you have a look at the documentation here https://lets-unify.ai/ivy/functional/ivy/layers/conv2d.html?highlight=conv2d you can see that `conv2d` actually does not take an `explicit_padding` argument. You are already passing that as `padding`. 
Same for below.

Reply:
I know that, but you said to add support for the `explicit_padding`, so i added the argument `explicit_padding` in `Conv2D `function

Reply:
so the Conv2D function should take the explicit_paddings in the padding variable itself, right

Reply:
if padding is "same" or "valid" ivy.conv2d gets the padding as padding and we ignore the explicit_paddings as it will be empty. If padding is explicit then ivy.conv2d get the explicit_paddings as padding.